fluentgen/src/main/java/com/azure/autorest/fluent/mapper/FluentLiveTestsMapper.java

```diff
@@ -0,0 +1,154 @@
+                            if (createMethod.isPresent()) {
+                                ResourceCreate create = createMethod.get();
+                                FluentResourceCreateExample createExample = ExampleParser.parseResourceCreate(resourceCollection, create, example);
+                                exampleMethod = fluentExampleTemplate.generateExampleMethod(createExample);
+                                setExampleStepFeatures(resultBuilder, testCaseBuilder, createExample, exampleMethod);
+                            } else {
+                                // find resourceUpdate
+                                Optional<ResourceUpdate> updateMethod = resourceCollection.getResourceUpdates().stream().filter(rc -> FluentUtils.exampleIsUpdate(rc.getMethodName()) && rc.getMethodName().equalsIgnoreCase(operation)).findFirst();
+                                if (updateMethod.isPresent()) {
+                                    ResourceUpdate update = updateMethod.get();
+                                    FluentResourceUpdateExample updateExample = ExampleParser.parseResourceUpdate(resourceCollection, update, example);
+                                    if (updateExample == null) {
+                                        return Optional.empty();
+                                    }
+                                    exampleMethod = fluentExampleTemplate.generateExampleMethod(updateExample);
+                                    setExampleStepFeatures(resultBuilder, testCaseBuilder, updateExample, exampleMethod);
+                                }
+                            }
+                        }
+                        if (exampleMethod != null) {
+                            resultBuilder.addHelperFeatures(testCaseBuilder.getHelperFeatures());
+                            return Optional.of(FluentExampleLiveTestStep.newBuilder().description(step.getDescription()).exampleMethod(exampleMethod).build());
+                        } else {
+                            // can't find method, ignore the whole test case altogether
+                            logger.warn(String.format("Operation : %s not found, ignore this test case.", operationId));
+                            return Optional.empty();
+                        }
+                    })
+                    .filter(Optional::isPresent)
+                    .map(Optional::get)
+                    .collect(Collectors.toList()));
+            return testCaseBuilder.build();
+        }).collect(Collectors.toList()));
+
+        return resultBuilder.build();
+    }
+
+    private String[] getOperationGroupPair(String operationId, CodeModel codeModel, FluentJavaSettings fluentJavaSettings) {
+        if (!operationId.contains("_")){
+            return new String[]{Utils.getNameForUngroupedOperations(codeModel, fluentJavaSettings), operationId};
+        }
+        return operationId.split("_");
+    }
```

Comment:
Return a static class with 2 variables. `String[]` is not good practice.

Also do a protection here or above. Some method (like pageNext) is not from swagger and hence null or empty operationId.

Reply:
Added.